src/main/java/seedu/savenus/storage/purchase/JsonAdaptedPurchase.java

```diff
@@ -1,4 +1,4 @@
-package seedu.savenus.storage;
+package seedu.savenus.storage.purchase;
```

Comment:
I think all things related to purchase should be kept within a purchase file in the directory though. Not sure what do you think? Seems more logical to put everything related to a field into one package.